doc/sphinx-guides/source/developers/making-releases.rst

```diff
@@ -70,11 +81,46 @@ Upload the following artifacts to the draft release you created:
 - installer (``cd scripts/installer && make``)
 - other files as needed, such as updated Solr schema and config files
 
-Publish Release
----------------
+Publish the Release
+-------------------
 
 Click the "Publish release" button.
 
+Close Milestone on GitHub and Create a New One
+----------------------------------------------
+
+You can find our milestones at https://github.com/IQSS/dataverse/milestones
+
+Now that we've published the release, close the milestone and create a new one.
+
+Note that for milestones we use just the number without the "v" (e.g. "5.10.1").
+
+Add the Release to the Dataverse Roadmap
+----------------------------------------
+
+Add an entry to the list of releases at https://www.iq.harvard.edu/roadmap-dataverse-project 
+
+Announce the Release on the Dataverse Blog
+------------------------------------------
+
+Make a blog post at https://dataverse.org/blog
+
+Announce the Release on the Mailing List
+----------------------------------------
+
+Post a message at https://groups.google.com/g/dataverse-community
+
+For Hotfixes, Merge Hotfix Branch into "develop" and Rename SQL Scripts
+-----------------------------------------------------------------------
+
+Note: this only applies to hotfixes!
+
+We've merged the hotfix into the "master" branch but now we need the fixes (and version bump) in the "develop" branch. Make a new branch off the hotfix branch and create a pull request against develop. Merge conflicts are possible and this pull request should go through review and QA like normal. Afterwards it's fine to delete this branch and the hotfix brach that was merged into master.
+
+Because of the hotfix version, any SQL scripts in "develop" should be renamed (from "5.11" to "5.11.1" for example). To read more about our naming conventions for SQL scripts, see :doc:`sql-upgrade-scripts`.
+
+Please note that version bumps and SQL script renaming both require all open pull requests to be updated with the latest from the "develop" branch so you might want to add any SQL script renaming to the hotfix branch before you put it through QA to be merged with develop. This way, open pull requests only need to be updated once.
```

Comment:
[possibly a dumb] Question - is this actually necessary, to go to the trouble of renaming flyway scripts on account of hotfix/emergency releases? In our current situation, we could have kept the existing V15.11.0.* scripts in the develop branch, and it wouldn't have broken anything, I don't think. Are there any situations where it would in fact be absolutely necessary?

Reply:
Renaming the SQL scripts is not strictly necessary but it keeps things tidy and consistent, which I think is worth it.

(That's why I wrote it this way and did the renaming.)

I say we try it. Try to be tidy and consistent. If it's really a pain point we can drop it some day.

Reply:
I can't imagine this ever becoming a real pain point. Hotfix releases are rare by nature. 
And sure, "tidy and consistent" should be a good enough reason.